tests/test_connections/test_ledger/test_ledger.py

```diff
@@ -442,6 +443,7 @@ async def test_not_hanging_with_multiplexer(self) -> None:
 
         # create a blocking task lasting `blocking_time` secs
         request, _ = self.create_ledger_dialogues()
+        request._sender = SOME_SKILL_ID
```

Comment:
the sender of the message is the skill, by default this is set to the connection. There probably is a better way to set this than like this.